### PR TITLE
Now its top level, update files in the repo a bit

### PR DESCRIPTION
**Pull request by ltratt**
Since there is now (e.g.) only one README, calling it `README.stats` isn't helpful. Change a couple of such files, and minimally update the README. Please see the latter commit for more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,40 +126,3 @@ are:
   * `--wallclock-only` to suppress most details
   * `--xlimits` and `--inset-xlimits` to "zoom in" on parts of the x-axis
 
-
-### Generating charts and plots without Krun
```
Comment (snim2) [inline README.md]: I think we still need this section?

Comment (ltratt) [reply README.md]: It's been moved and renamed a bit.

File: README.md [hunk 2]
```diff
@@ -56,44 +89,27 @@ tabling or plotting scripts.
...
-benchmarks (e.g. Da Capo, Octane, etc.).
-
-Both script generate LaTeX code which needs to be translated to PDF (e.g. with
-`pdflatex`). The largest difference between the two scripts is that `_main`
-produces tables where every row is a VM (and benchmarks are grouped in blocks),
-and `_other` produces tables where every row is a benchmark (and benchmarks may
-be grouped by VMs, if more than one VM has been benchmarked).
-
-The two scripts are invoked slightly differently:
+`bin/table_classification_summaries_others` plots summary tables with various
```
Comment (snim2) [inline README.md]: Users who started with `.csv` input should use `warmup_stats` to produce tables and plots, users who started with Krun output should use the `table_` script.

**Comment by ltratt** [comment]
OK, that hopefully addresses (albeit minimally) the comments.

**Comment by snim2** [comment]
OK, do you want to squash?

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**